### DISCUSSION TITLE
XDS load on IstioD with Ambient Mesh

### DISCUSSION THREAD

**Discussion by nirvanagit** [istio/istio#52857] [Q&A]
Given Ambient MESH uses ZTunnel and WayPoint proxies, how do they, specially WayPoint proxies get information about EDS?

Given the same workload, will moving to Ambient from Sidecar result in reduced load on IstioD?

Basically, I am trying to gauge the work IstioD would do when using Ambient.

**Comment by howardjohn** [comment]
https://www.youtube.com/watch?v=S39yo6ZJ4iM covers this in depth. Waypoints still use the standard Envoy types, but ztunnel uses its own optimized type.

I have been unable to get Istiod meaningfully loaded with the new ambient model for the most part. Obviously you can apply 1000s of routes on a waypoint and have the same load as if you had done that with sidecar/ingress, but its quite an uncommon usage. For ztunnel, even with O(200k) pods istiod is barely working in our testing.
